uaclient/entitlements/repo.py

```diff
@@ -92,6 +92,10 @@ def _perform_enable(self) -> bool:
         @return: True on success, False otherwise.
         @raises: UserFacingError on failure to install suggested packages
         """
+        apt.setup_apt_proxy(
```

Comment:
I think we could move this call into `setup_apt_config`. But this is just a nit, feel free to change it or not